modules/artifact-activation/src/test/java/org/mule/runtime/module/artifact/activation/internal/deployable/LightweightDeployableProjectModelBuilderTestCase.java

```diff
@@ -6,169 +6,152 @@
  */
 package org.mule.runtime.module.artifact.activation.internal.deployable;
 
-import static org.mule.runtime.core.api.util.FileUtils.cleanDirectory;
+import static org.mule.maven.client.api.MavenClientProvider.discoverProvider;
+import static org.mule.runtime.globalconfig.api.GlobalConfigLoader.getMavenConfig;
+import static org.mule.runtime.globalconfig.api.GlobalConfigLoader.reset;
 import static org.mule.runtime.module.artifact.activation.internal.MavenTestUtils.installArtifact;
-import static org.mule.test.allure.AllureConstants.ClassloadingIsolationFeature.CLASSLOADING_ISOLATION;
-import static org.mule.test.allure.AllureConstants.ClassloadingIsolationFeature.ClassloadingIsolationStory.ARTIFACT_DESCRIPTORS;
+import static org.mule.test.allure.AllureConstants.DeploymentTypeFeature.DEPLOYMENT_TYPE;
+import static org.mule.test.allure.AllureConstants.DeploymentTypeFeature.DeploymentTypeStory.LIGHTWEIGHT;
 
-import static java.lang.Math.random;
-import static java.nio.file.Files.createTempDirectory;
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singletonList;
 
-import static org.apache.commons.io.FileUtils.deleteQuietly;
-import static org.hamcrest.Matchers.contains;
-import static org.hamcrest.Matchers.containsInAnyOrder;
-import static org.hamcrest.Matchers.equalTo;
-import static org.hamcrest.Matchers.hasProperty;
+import static org.hamcrest.CoreMatchers.is;
 import static org.junit.Assert.assertThat;
 
+import org.mule.maven.client.api.MavenReactorResolver;
+import org.mule.maven.client.api.model.BundleDescriptor;
 import org.mule.runtime.module.artifact.activation.api.deployable.DeployableProjectModel;
 import org.mule.runtime.module.artifact.activation.internal.maven.LightweightDeployableProjectModelBuilder;
 import org.mule.runtime.module.artifact.api.descriptor.BundleDependency;
+import org.mule.tck.junit4.AbstractMuleTestCase;
 import org.mule.tck.junit4.rule.SystemProperty;
 
 import java.io.File;
-import java.io.IOException;
 import java.net.URISyntaxException;
+import java.util.List;
+import java.util.Optional;
 
-import io.qameta.allure.Description;
 import io.qameta.allure.Feature;
 import io.qameta.allure.Issue;
 import io.qameta.allure.Story;
-import org.junit.After;
-import org.junit.AfterClass;
-import org.junit.ClassRule;
+import org.junit.Before;
+import org.junit.Rule;
 import org.junit.Test;
 
-@Feature(CLASSLOADING_ISOLATION)
-@Story(ARTIFACT_DESCRIPTORS)
-@Issue("W-12069164")
```

Comment:
is it ok to remove this `@Issue` here?

Reply:
I moved those tests to https://github.com/mulesoft/mule/pull/12016/files/f16ad2a26eaca7161e431c1785556c30c253bf57#diff-bf0ad39083fc3fdd186c51c57bf93ad6579af222998b2a2eadba524c8a44ba08R44